.github/workflows/php-coding-standards.yml

```diff
@@ -0,0 +1,47 @@
+name: PHP Coding Standards
+
+on:
+  workflow_call:
+    inputs:
+      php:
+        default: 8.0
+        required: false
+        type: number
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    name: phpcs ${{ inputs.php }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Get Composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Set up Composer caching
+        uses: actions/cache@v3
+        env:
+          cache-name: cache-composer-dependencies
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ inputs.php }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ inputs.php }}-composer-
```

Comment:
I wonder why you are not using https://github.com/ramsey/composer-install or https://github.com/php-actions/composer here.

Is there a reason?